app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [my_favourite_films,
+                                marks, collection_of_coins],
+                    "immutable": [lucky_number, pi,
+                                  one_is_a_prime_number, name, profile_info]}
```

Comment:
You need to correct the code style, I'll show you an example:

sorted_variables = {
    "mutable": [my_favourite_films, marks, collection_of_coins],
    "immutable": [lucky_number, pi,one_is_a_prime_number, name, profile_info]
}
You can do the same with a list inside a dictionary if the variables in it are long by name.